src/main/java/com/google/codeu/servlets/UserListServlet.java

```diff
@@ -1,18 +1,32 @@
 package com.google.codeu.servlets;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
+import java.util.Set;
 
 @WebServlet("/user-list")
 public class UserListServlet extends HttpServlet {
+    private Datastore datastore;
 
     @Override
-    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    public void init() {
+        datastore = new Datastore();
+    }
 
-        response.getOutputStream().println("this will be my user list");
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        
+        Set<String> users=datastore.getUsers();
```

Comment:
nit: You don't need users, could just do `for (String user:datastore.getUsers())`

Reply:
Thanks!, Changed this too.